playgroundFaq.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: subpage
+title: "CHAPI Playground Integration Guide"
+permalink: /developers/playgroundfaq
+---
+# CHAPI Playground Integration Guide
+
+Have you already integrated CHAPI into your project and now you’re interested in making your Issuer, Wallet, or Verifier available for experimentation? This Integration Guide/FAQ for the CHAPI Playground will help you accomplish this!
+
+## Issuers
+### New issuer introduction:
+You can integrate your Issuer into our tooling with the [CHAPI-Playground Issuer](https://playground.chapi.io/issuer).  
+
+If you can provide a [VC-API /issue endpoint](https://w3c-ccg.github.io/vc-api/#issue-credential), we can add your issuer to the list of options accessible from the "Gear" icon on the site.  That should be all we need for integration with the Veres Wallet.  (Note that you'll also be able to use this site to demonstrate issuance with any other CHAPI-enabled wallets).
```

Comment:
Please generalize the last two sentences... maybe something link - "This will allow your issuer to integrate with any CHAPI-enabled wallet."  (and eliminate reference to Veres).